### ISSUE TITLE
Simplify accumulo-testing command and create Docker image

### ISSUE THREAD

**Issue by mikewalch** [apache/accumulo-testing#42]
It would be great if accumulo-testing could be simplified and run in Docker.  Below is possible usage.

```
$ ./bin/accumulo-testing 

Usage: accumulo-testing <test> (<argument>)

Available tests:

   ci <application>    Runs continous ingest <application>.
                                Possible applications: createtable, ingest, walk, batchwalk, scan, verify, moru

   rw <module>         Runs random walk <module>
                                Modules located in core/src/main/resources/randomwalk/modules
```

**Comment by mikewalch** [comment]
Simplifying the command above would involve removing the ability to run in YARN using Twill.  Running multiple test clients on cluster would be done using the Docker image that the project creates and cluster management tool like Docker Swarm, Kubernetes or Mesos/Marathon.

**Comment by mikewalch** [comment]
Fixed by #43
